### DISCUSSION TITLE
End of life each version

### DISCUSSION THREAD

**Discussion by AxelHumbert** [pallets/flask#4909] [Q&A]
Hello,

I would like to know if you have a table or an other thing to resume the end of life of each version ?
Maybe, when a new version is released, the support of the latest version is stopped. 
Could you confirm me ?

Best regards

Axel HUMBERT

**Comment by ViniciusMarchi** [comment]
As far as I know everything about versioning is published in the [changelog](https://flask.palletsprojects.com/en/2.2.x/changes/), I don't know if it helps you, but I hope so

**Reply by AxelHumbert** [reply]
Thank you ViniciusMarchi but I didn't find my answer in the changelog.

Maybe, I was not clear in my request sorry. I want to understand when a version is not supported. 
For example, we have now the 2.X version, so do I consider that the 1.X version support is stopped ?

**Reply by tachyondecay** [reply]
[Flask does not use semantic versioning](https://github.com/pallets/flask/issues/4455#issuecomment-1044527327).

Only the most recent release is supported. For Flask, at the time of this comment, that is version 2.2.2. When version 2.2.3 is released, it will be the only supported version.
